apps: added new fluentd buffer alerts (#1107)
elastisys/compliantkubernetes-apps: WIP-CHANGELOG.md

### Release notes

### Updated

### Changed

### Fixed

### Added

> - a new alert `FluentdAvailableSpaceBuffer` to get notified when the fluentd buffer gets full

Reviewer: ```suggestion
- a new alert `FluentdAvailableSpaceBuffer`, notifies when the fluentd buffer is filling up
```
